waku/discovery/waku_discv5.nim

```diff
@@ -108,6 +108,19 @@ proc new*(
     topicSubscriptionQueue: queue,
   )
 
+proc updateAnnouncedMultiAddress*(
+    wd: WakuDiscoveryV5, addresses: seq[MultiAddress]
+): Result[void, string] =
+  let encodedAddrs = multiaddr.encodeMultiaddrs(addresses)
+
+  wd.protocol.updateRecord([(MultiaddrEnrField, encodedAddrs)]).isOkOr:
+    return err("failed to update multiaddress in ENR: " & $error)
+
+  debug "ENR updated successfully with new multiaddress",
+    enr = wd.protocol.localNode.record.toUri(), record = $(wd.protocol.localNode.record)
```

Comment:
Maybe we should change the keys to be consistent with the logs at `updateENRShards` and `start` (or viceversa)

```suggestion
    enrUri = wd.protocol.localNode.record.toUri(), enr = $(wd.protocol.localNode.record)
```